lib/src/model/directus_user.dart

```diff
@@ -43,4 +43,19 @@ class DirectusUser {
 
   DirectusUser.from({required this.id, required String email})
       : allProperties = {"id": id, "email": email};
+
+  String get fullName {
+    final String currentFirstName = (firstname == null) ? "" : firstname!;
```

Comment:
I'd rather avoid using forced unwrap (usage of ! for null values). It might lead to crashes in multithreaded contexts.
You can use a local variable to test and unwrap safely : 
```dart
final String currentFirstName = firstname;
return currentFirstName ?? "";
```